### ISSUE TITLE
BUG: FAB Causes a crash on android

### ISSUE THREAD

**Issue by justJay-dev** [yamankatby/react-native-material#34]
Using the example code in the docs, on `1.3.7` 

Pixel 5 android 12.

Is it cool if I patch this?

```
Error: Attempting to run JS driven animation on animated node that has been moved to "native" earlier by starting an animation with `useNativeDriver: true`
at node_modules/react-native/Libraries/Core/ExceptionsManager.js:104:6 in reportException
at node_modules/react-native/Libraries/Core/ExceptionsManager.js:172:19 in handleException
at node_modules/react-native/Libraries/Core/setUpErrorHandling.js:24:6 in handleError
at node_modules/expo-error-recovery/build/ErrorRecovery.fx.js:12:21 in ErrorUtils.setGlobalHandler$argument_0
```

**Comment by justJay-dev** [comment]
tl;dr you pretty much have to either always `useNativeDriver` or never

**Comment by justJay-dev** [comment]
I can't get the example app to build with local changes, but yeah, what you want to do is find instances of `useNativeDriver` and set it to true always based on a given Platform.OS or just set it to true always, or false always, it doesn't matter, but you can't mix. 


May be worth leaving some notes on your local setup, the example app just pulls from the latest published build. Lost some time to that, cheers!

**Comment by yamankatby** [comment]
Hi Jay,
Thank you for reporting this issue. Hopefully, I will find some time to fix it this week.

> If you have time and would like to help please make a PR. I'm not sure why the example app doesn't build on your computer. I have used [create-react-native-library](https://github.com/callstack/react-native-builder-bob) to create this library.

**Comment by justJay-dev** [comment]
Got a PR open :) https://github.com/yamankatby/react-native-material/pull/35

**Comment by rhalaly** [comment]
Any estimation of when the PR will be merged?

**Comment by arturfri** [comment]
I'm still having this issue on version 1.3.7. Apparently the merge that was supposed to deploy this fix wasn't approved by Vercel

**Comment by souzaluiz** [comment]
@yamankatby still happening in version 1.3.7
